x/auth/account.go

```diff
@@ -345,6 +347,17 @@ func (bva *BaseVestingAccount) TrackUndelegation(amount sdk.Coins) {
 	}
 }
 
+// ToBaseVestingAccount converts a vesting account to a BaseVestingAccount. This
+// can be used to retrieve common fields that all vesting accounts share.
+func (bva BaseVestingAccount) ToBaseVestingAccount() BaseVestingAccount {
```

Comment:
Alternatively, we can simply add getters for `OriginalVesting`, `DelegatedFree`, and `DelegatedVesting` to the interface.

Reply:
yeah let's do that, and also couldn't we just `return bva` here?

Reply:
Yes, we could :)